mqtt/mqtt-broker/src/settings/size.rs

```diff
@@ -257,7 +257,7 @@ mod tests {
         }
 
         #[test]
-        fn it_cannot_parse_input(input in r".*[0-9]{9}.*(k|K|m|M|g|G)?.*") {
+        fn it_cannot_parse_input(input in r"[^0-9]+[0-9]{9}[^0-9]+(k|K|m|M|g|G)?.*") {
```

Comment:
Was this validated with another regex validator to double check it matches only what we expect? Would it be worthwhile to have this kind of test be fuzzed?

Reply:
I did a quick check of the regex, and at least I was not able to repro the issue with the previously reported failed value. I believe it is very tiny feature to warrant fuzz tests.